approve.py

```diff
@@ -241,6 +244,11 @@ def pause_error_msg():
             print()
             pause_error_msg()
             continue
+       
+        if account.get('dry_run'):
```

Comment:
you don't need `.get`, you already asserted above that the key exists; just `account['dry_run']` will do